src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -364,6 +364,7 @@ dellsc_volume_folder=cmpl_iscsi_volumes
 
 # Unity cinder parameters. 
 enable_unity_backend=false
+cinder_unity_container_version=latest
```

Comment:
Can we add a comment here?  Something like, "In most cases, "latest" will work fine.  If you need to change to a specific version, specify it here.